### DISCUSSION TITLE
Incremential IDs using xid, ksuid (or equivalent)

### DISCUSSION THREAD

**Discussion by brunobord** [zk-org/zk#325] [Ideas]
### Verify

- [X] I searched the existing discussions for similar ideas

### Share your idea or feature request

From the documentation:

> Sequential (incremented) IDs are currently not supported by zk.

[source](https://github.com/mickael-menu/zk/blob/main/docs/note-id.md)

Have you ever considered using identifiers that look like `uuid`s but are sortable?

I was thinking something similar to [xid](https://github.com/rs/xid) or [ksuid](https://github.com/segmentio/ksuid) (written in Go, how fortunate).

**Comment by mickael-menu** [comment]
You can already use it if you provide the output to the `id` parameter.

```shell
go install github.com/segmentio/ksuid/cmd/ksuid@latest
zk new --id `ksuid`
```

Otherwise I would be fine merging this addition in the repo if someone wants to contribute it. Personally I have no use for sequential IDs.
